tests/fixtures/apidoc/luna_hostgroup.json

```diff
@@ -0,0 +1 @@
+luna.json
```

Comment:
@evgeni is this really the luna setup?
I know we already have a special api.json for snapshot. Maybe we want one for openscap?

Reply:
I didn't look if the JSON is from a Luna setup, if that's what you're asking.

My original suggestion to use luna here was to allow us to add `foreman_ansible` related hostgroup settings easily in the future. *Today* a Foreman+OpenSCAP setup/JSON would suffice. But that's also harder to produce than `vagrant up luna` :)

Reply:
Yes, the question is (and i guess to @bagasse then) is this produced from `vagrant up luna`?

Reply:
Yes, that's vagrant up luna ;)

Reply:
This renaming should not be needed in any case. That is why we do all that extra matching dance in the vcr_wrapper.